src/utils/openInBrowser.ts

```diff
@@ -21,5 +22,17 @@ export const openInBrowser = (url: string, internal = true) => {
     return Navigation.handleAction(Routes.DAPP_BROWSER_SCREEN, { url });
   }
 
+  if (SafariContext) {
+    const handleOpenInSafariContext = async (url: string) => {
+      try {
+        await WebBrowser.openBrowserAsync(url);
+      } catch (error) {
+        // This error is thrown when the user closes the browser, so we can safely ignore it.
+        logger.debug('[AddCash]: Expo WebBrowser closed by user', { message: (error as Error).message });
```

Comment:
Based on the docs, I don't think this is the case. The function should probably just return the promise like it does for the other cases:

`return WebBrowser.openBrowserAsync(url)`

<img width="1071" alt="Screenshot 2025-06-11 at 12 21 02 PM" src="https://github.com/user-attachments/assets/51ca21e8-56bd-4a5e-abeb-80cd297dbebe" />